velox/docs/functions/spark/map.rst

```diff
@@ -0,0 +1,34 @@
+===========================
+Map Functions
+===========================
+
+.. function:: getmapvalue(map(K,V), key) -> V
+    :noindex:
+
+    Returns value for given ``key``, or ``NULL`` if the key is not contained in the map.
+
+.. function:: element_at(map(K,V), key) -> V
+    :noindex:
+
+    Returns value for given ``key``, or ``NULL`` if the key is not contained in the map.
+
+.. function:: map() -> map<unknown, unknown>
+
+    Returns an empty map. ::
+
+        SELECT map(); -- {}
+
+.. function:: map(array(K), array(V)) -> map(K,V)
+
+    Returns a map created using the given key/value arrays. ::
```

Comment:
Perhaps, document the behavior in the presence of duplicate keys.

Reply:
added "Duplicate map key will cause exception."